app/services/follow_recommendations_service.rb

```diff
@@ -83,14 +82,16 @@ def build_follow_graph(indirect_follows, direct_follow_ids)
     indirect_follow_map
   end
 
-  # Returns an array of default follows in the same JSON format as the public API using AccountSerializer
+  # Returns an array of default follows from V2 Suggestions API
   def generate_default_follows
-    # domain: nil makes sure we're looking for local accounts
-    accounts = Account.where(username: DEFAULT_FOLLOW_LIST, domain: nil)
-    serializer = ActiveModel::Serializer::CollectionSerializer.new(
-      accounts, serializer: REST::AccountSerializer
-    )
-    JSON.parse(serializer.to_json)
+    account = account_from_handle
+    AccountSuggestions.get(account, MAX_RESULTS).map { |suggest| suggest.account.acct }
+  end
```

Comment:
here we reuse the Mastodon `AccountSuggestions` and return those handles rather than the previous `(bart, mark, tsepiso)` from the past